### DISCUSSION TITLE
Elementary 7.1 Horus — can't disable NUMLOCK by default

### DISCUSSION THREAD

**Discussion by abambresmiguel** [elementary/triage#440] [Q&A]
Hello! I got a laptop with Elementary OS 7.1 Horus installed, and cant find a way to disable NUMLOCK by default. Everytime I login i have to manually disable NUMLOCK :(

Could you possibly help me, please 🙏🏻?

(didnt find any clue on the web)

**Comment by jeremypw** [comment]
You *may* be able to set the numlock state at startup by going into the BIOS editor (usually by pressing a certain key early in the startup process - dependent on your machine).  What worked for me was to set the setting `org.gnome.desktop.peripherals.keyboard remember-numlock-state` to `true`.  Then when you log in, the numlock state is restored to how you left it.   You can change the setting using "Dconf Editor" (available in the App Centre) or just by running `gsettings set org.gnome.desktop.peripherals.keyboard remember-numlock-state true` in a terminal.  This will only effect the individual user - not the greeter.

**Reply by abambresmiguel** [reply]
Thanks so much Jeremy. Back then i tried the BIOS but didnt find any option. Will try the terminal first. Be well.

**Reply by abambresmiguel** [reply]
Unfortunatelly didnt work for me.... every finished login coming from a start or restart of the OS makes numlock ON... Note that **during** login numlock is always disabled by default. Thanks for your efforts.

**Reply by jeremypw** [reply]
Sorry it didn't work. If you turn `NUMLOCK` off while logged in then log out it *should* remember the off state and reapply it when you log back in.  This is what happens for me. I wonder if some other software/application is turning it back on for some reason :shrug:  Did you install anything else?  It sounds like the hardware default is for it to be off, so something must be actively turning it on.

**Reply by abambresmiguel** [reply]
So far have only installed the Brave browser... i got the new laptop with Elementary OS 7.1 Horus already installed. The company i bought it from cant solve it either. Never mind, i will be able to live with that little issue until some OS update solves it. **Thanks so much**, Jeremy.

**Reply by abambresmiguel** [reply]
Have just filed an issue on https://github.com/elementary/switchboard-plug-keyboard -- that was where the feedback app on my system led me to. Lets see.

**Reply by jeremypw** [reply]
I hope someone can help you.

**Reply by abambresmiguel** [reply]
Its solved. Today, somehow, when i powered it on the issue had gone lol. **thanks**

**Reply by abambresmiguel** [reply]
I m now sure of **what caused it**:

having the wireless mouse connected, regardless the usb port, **before** powering on or restarting the machine.
